packages/core/src/services/issues/merge.ts

```diff
@@ -1,19 +1,482 @@
+  anchorIssue,
+  candidates,
+  totals,
+}: {
+  anchorIssue: Issue
+  candidates: Issue[]
+  totals: Map<number, number>
+}) {
+  const winner = candidates.reduce((current, issue) => {
+    const currentTotal = totals.get(current.id) ?? 0
+    const nextTotal = totals.get(issue.id) ?? 0
+
+    if (nextTotal > currentTotal) return issue
+    if (nextTotal < currentTotal) return current
+
+    if (issue.id === anchorIssue.id) return issue
+    if (current.id === anchorIssue.id) return current
+
+    return issue.id < current.id ? issue : current
+  }, candidates[0]!)
+
+  const mergedIssues = candidates.filter((issue) => issue.id !== winner.id)
+
+  return { winner, mergedIssues }
+}
+
+function mergeIssueCentroids({
+  winner,
+  mergedIssues,
+  timestamp,
+}: {
+  winner: Issue
+  mergedIssues: Issue[]
+  timestamp: Date
+}): IssueCentroid | undefined {
+  const centroids = [winner, ...mergedIssues]
+    .filter((issue) => issue.centroid.base.length > 0)
+    .map((issue) => ({ ...issue.centroid, updatedAt: issue.updatedAt }))
+
+  if (centroids.length === 0) return undefined
+  if (centroids.length === 1) {
+    return { base: centroids[0]!.base, weight: centroids[0]!.weight }
+  }
+
+  return mergeCentroids(centroids, timestamp)
+}
+
+async function linkEvaluationResults({
+  workspaceId,
+  winnerId,
+  issueIds,
+  timestamp,
+  tx,
+}: {
+  workspaceId: number
+  winnerId: number
+  issueIds: number[]
+  timestamp: Date
+  tx: typeof database
+}) {
+  if (issueIds.length === 0) return
+
+  const associations = await tx
+    .select({
+      evaluationResultId: issueEvaluationResults.evaluationResultId,
+    })
+    .from(issueEvaluationResults)
+    .where(
+      and(
+        eq(issueEvaluationResults.workspaceId, workspaceId),
+        inArray(issueEvaluationResults.issueId, issueIds),
+      ),
+    )
+
+  if (associations.length === 0) return
+
+  await tx
+    .insert(issueEvaluationResults)
+    .values(
+      associations.map(({ evaluationResultId }) => ({
+        workspaceId,
+        issueId: winnerId,
+        evaluationResultId,
+        createdAt: timestamp,
+        updatedAt: timestamp,
+      })),
+    )
+    .onConflictDoNothing()
+
+  await tx
+    .update(evaluationResultsV2)
+    .set({ issueId: winnerId, updatedAt: timestamp })
+    .where(
+      and(
+        eq(evaluationResultsV2.workspaceId, workspaceId),
+        inArray(evaluationResultsV2.issueId, issueIds),
+      ),
+    )
+}
+
+async function relinkEvaluations({
+  workspaceId,
+  winnerId,
+  issueIds,
+  timestamp,
+  tx,
+}: {
+  workspaceId: number
+  winnerId: number
+  issueIds: number[]
+  timestamp: Date
+  tx: typeof database
+}) {
+  if (issueIds.length === 0) return
+
+  await tx
+    .update(evaluationVersions)
+    .set({ issueId: winnerId, updatedAt: timestamp })
+    .where(
+      and(
+        eq(evaluationVersions.workspaceId, workspaceId),
+        inArray(evaluationVersions.issueId, issueIds),
+      ),
+    )
+}
```

Comment:
I don't think we want to do this. I think we talk about these evaluations of the no winners. 

1. Set `live` to false
2. Add a new column called `ignoredAt` and filter these ignored evaluations from the UI

Talk with @cesr or @learningbizz I could be wrong

Reply:
What was the conclusion here?